src/tools/immutable_object_cache/CacheServer.h

```diff
@@ -39,7 +38,6 @@ class CacheServer {
   stream_protocol::endpoint m_local_path;
   stream_protocol::acceptor m_acceptor;
   uint64_t m_session_id = 1;
```

Comment:
I suppose now that we don't need a map to see which session_id corresponds to which CacheSession, we can just ditch this?

Reply:
yes, later will remove these variable. many thanks for your review ~